### DISCUSSION TITLE
Support additional social causes on the home page?

### DISCUSSION THREAD

**Discussion by neaucode** [mdx-js/mdx#1550] [Ideas]
Hello, I noticed the BLM banner and was wondering if mdx is open to supporting lesser known social causes on the home page!

I know BLM is a big trend right now, but there are many social injustices in the world and it would be great to use this code repository for more visibility. Does anyone have thoughts on the matter?

**Comment by neaucode** [comment]
Looks like ceasefire is the new thing, maybe it's a good marketing platform (I'm not one to judge)

**Comment by ChristianMurphy** [comment]
The maintainer team may support future causes.
That said we're not a marketing platform please don't spam the team with marketing requests for random causes.
